mars/deploy/oscar/local.py

```diff
@@ -51,6 +51,9 @@ def __init__(self: ClusterType,
                  subprocess_start_method: str = None,
                  config: Union[str, Dict] = None,
                  web: Union[bool, str] = 'auto'):
+        # load config file to dict.
+        if not config or isinstance(config, str):
+            config = load_config(config)
```

Comment:
Why do not you load config within `create_supervisor_actor_pool` and `create_worker_actor_pool`?

Reply:
The ray backend needs to load the config before creating pools. Another problem is that the `create_xxx_actor_pool` APIs accept modules instead of config.